packages/delance/package.yaml

```diff
@@ -0,0 +1,20 @@
+---
+name: delance
+description: þe Lance of Python language server
+homepage: https://sr.ht/~self/delance
+licenses:
+  - 0BSD
+  - MIT
+languages:
+  - Python
+categories:
+  - LSP
+
+source:
+  id: pkg:npm/@delance/runtime@2024.3.100
+
+schemas:
+  lsp: https://raw.githubusercontent.com/microsoft/pyright/main/packages/vscode-pyright/schemas/pyrightconfig.schema.json
```

Comment:
This seems to be a configuration schema for pyright. I couldn't find one for delance so removing
```suggestion
```